<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QMainWindow" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Начальная точка</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="start_latitude_line">
          <property name="placeholderText">
           <string>Широта</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="start_latitude_hemisphere_box">
          <item>
           <property name="text">
            <string>с. ш.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ю. ш.</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="start_longitude_line">
          <property name="placeholderText">
           <string>Долгота</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="start_longitude_hemisphere_box">
          <item>
           <property name="text">
            <string>в. д.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>з. д.</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Искомая точка</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="assigned_latitude_line">
          <property name="placeholderText">
           <string>Широта</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="assigned_latitude_hemisphere_box">
          <item>
           <property name="text">
            <string>с. ш.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ю. ш.</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>290</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Курс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="course_line">
        <property name="placeholderText">
         <string>Курс</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="calculate_longitude_button">
      <property name="text">
       <string>Вычислить долготу второй точки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Долгота искомой точки</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="unknown_longitude_line">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="unknown_longitude_hemisphere_box">
        <item>
         <property name="text">
          <string>в. д.</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>з. д.</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Длина локсодромы</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="loxodrome_length_line">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="units_of_loxodrome_measurement_box">
            <item>
             <property name="text">
              <string>км</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>морские мили</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>сухопутные мили</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Длина ортодромы</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="orthodrome_length_line">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="units_of_orthodrome_measurement_box">
            <item>
             <property name="text">
              <string>км</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>морские мили</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>сухопутные мили</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Что ищем?</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Курс по начальной и конечной точкам</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Долготу по начальной точке и курсу</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
